images/ntia-conformance-checker/tests/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "check-ply" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME --file /var/lib/db/sbom/py3-ply-3.11_git20180215-r2.spdx.json"
```

Comment:
Nice.

Reply:
It would be if it passed 😉 

I'm expecting this to fail, then I'll make the test pass and we can iterate with the resulting image 🤞